Documentation/installation/requirements-aks.rst

```diff
@@ -7,8 +7,14 @@ BYOCNI (``none``)             Encapsulation   Cluster Pool        Kubernetes CRD
 Legacy Azure IPAM (``azure``) Direct Routing  Azure IPAM          Kubernetes CRD
 ============================= =============== =================== ==============
 
-Using `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_
-is the preferred way to run Cilium on `Azure Kubernetes Service (AKS) <https://docs.microsoft.com/en-us/azure/aks/>`_,
-however integration with the Azure stack via the :ref:`Azure IPAM<ipam_azure>`
-is not available and will only work with clusters not using BYOCNI. While still
-maintained for now, this mode is considered legacy.
+The preferred way to run Cilium on `Azure Kubernetes Service (AKS) <https://docs.microsoft.com/en-us/azure/aks/>`_ is 
+either `Bring your own CNI <https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli>`_ or 
+`Azure CNI Powered by Cilium <https://learn.microsoft.com/en-us/azure/aks/azure-cni-powered-by-cilium>`_. 
+Choosing between these options depends on your specific requirements and use case. 
+If you need full control over Cilium and its features, then Bring your own CNI approach might be preferred. 
+On the other hand, if you prefer a managed solution with simplified installation and 
+integration with Azure Virtual Network stack, then Azure CNI Powered by Cilium option might be suitable.
+
+Another way to install Cilium on AKS is through chaining with Azure CNI :ref:`chaining_azure` and 
+can be integrated with Azure Stack via :ref:`Azure IPAM<ipam_azure>`. However this integration will not work for clusters using BYOCNI. 
+While it is still maintained for now, it is considered a legacy mode.
```

Comment:
I'd be inclined to drop this part. Users who need this already know about it and are using it, and they can track down the specific docs page for this information. I don't think it's particularly valuable to point new users towards these modes.